scripts/run-integration-tests.sh

```diff
@@ -30,4 +30,9 @@ for ((i = "$agent_number"; i <= "${#test_dirs[@]}"; )); do
 done
 
 printf "\nRunning the following test suites:\n\n%s\n\nStarting tests...\n\n" "$(echo "${dirs[@]}" | tr -s ' ' '\n')"
-ginkgo -keepGoing --slowSpecThreshold 60 "${dirs[@]}"
+
+if [ -z "${PKCS11_LIB-}" ] || [ -z "${PKCS11_LABEL-}" ] || [ -z "${PKCS11_PIN-}" ]; then
+   ginkgo -keepGoing --slowSpecThreshold 60 "${dirs[@]}"
+else
+   ginkgo -tags pkcs11 -keepGoing --slowSpecThreshold 60 "${dirs[@]}"
```

Comment:
Why would the tags be here instead of passed as arguments to the build server? Combined with the above check we wouldn't be building with pkcs11 tags if we're not running the tests.